flask_restalchemy/resources/utils.py

```diff
@@ -114,13 +121,22 @@ def build_filter_operator(column_name, request_filter, serializer):
         limit = request.args['limit']
         query = query.limit(limit)
     if 'order_by' in request.args:
-        col = getattr(model,request.args['order_by'])
-        if 'desc' in request.args:
-            query = query.order_by(desc(col))
-        else:
-            query = query.order_by(col)
-    if 'page' not in request.args:
-        resources = query.all()
-    else:
-        resources = query.paginate()
-    return resources
+        fields = request.args['order_by'].split(',')
+        for field in fields:
+            if field[0] == '-':
+                col = getattr(model, field[1:])
+                query = query.order_by(desc(col))
+            else:
+                col = getattr(model, field)
+                query = query.order_by(col)
+    if 'page' in request.args:
+        data = query.paginate()
+        return {
+            'page': data.page,
+            'per_page': data.per_page,
+            'count': data.total,
+            'results': [model_serializer.dump(item) for item in data.items]
+        }
+
```

Comment:
I would use an "else" here so the above **return** would not be confused with a [guard clause](https://refactoring.com/catalog/replaceNestedConditionalWithGuardClauses.html)

Reply:
OK